proposals/lip-0041.md

```diff
@@ -175,15 +175,14 @@ During the genesis state initialization stage, the following steps are executed.
 
 Let `genesisBlockAssetBytes` be the bytes included in the block assets for the Auth module and let `genesisBlockAssetObject` be the deserialization of `genesisBlockAssetBytes` according to the `genesisAuthStoreSchema` schema, given below. 
 
-* Check that `genesisBlockAssetObject.authDataSubstore.storePrefix` equals `STORE_PREFIX_AUTH`.
-* Check that every entry `authAccount` in `genesisBlockAssetObject.authDataSubstore.storeEntries` has a different `authAccount.storeKey`.
-* For each entry `authAccount` in `genesisBlockAssetObject.authDataSubstore.storeEntries`, if `authAccount.storeValue.numberOfSignatures` is not 0 (a multisignature account), check that the rules listed in [LIP 0017](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0017.md#multisignature-account-registration) are respected. In particular:
-  * The keys in the `authAccount.storeValue.mandatoryKeys` array are pairwise distinct and ordered lexicographically. The array may be empty.
-  * The keys in the `authAccount.storeValue.optionalKeys` array are pairwise distinct and ordered lexicographically. The array may be empty.
+* Check that every entry `authAccount` in `genesisBlockAssetObject.authDataSubstore` has a different `authAccount.storeKey`.
+* For each entry `authAccount` in `genesisBlockAssetObject.authDataSubstore`, if `authAccount.storeValue.numberOfSignatures` is not 0 (a multisignature account), check that the rules listed in [LIP 0017](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0017.md#multisignature-account-registration) are respected. In particular:
+  * The keys in the `authAccount.storeValue.mandatoryKeys` array are pairwise distinct and in lexicographical order. The array may be empty.
```

Comment:
Mentioned in LIP17, but if you wanted, we could add: "All elements of `mandatoryKeys` and `optionalKeys` have length 32."

Reply:
fixed